Sources/NIOPosix/System.swift

```diff
@@ -153,6 +153,9 @@ private let sysKevent = kevent
 private let sysSendMmsg: @convention(c) (CInt, UnsafeMutablePointer<CNIODarwin_mmsghdr>?, CUnsignedInt, CInt) -> CInt = CNIODarwin_sendmmsg
 private let sysRecvMmsg: @convention(c) (CInt, UnsafeMutablePointer<CNIODarwin_mmsghdr>?, CUnsignedInt, CInt, UnsafeMutablePointer<timespec>?) -> CInt = CNIODarwin_recvmmsg
 #endif
+#if canImport(Darwin) || os(Linux)
```

Comment:
`ioctl` should be available ~everywhere? If Windows doesn't have it maybe `#if !os(Windows)`?

Reply:
Right, again, I was avoiding adding things on platforms I wasn't testing the build on.

Looking now it seems the spelling is slightly different in e.g. Musl:

```c
int ioctl (int, int, ...);
```
(source: https://git.musl-libc.org/cgit/musl/tree/include/sys/ioctl.h)

cf.

```c
int ioctl(int, unsigned long, ...);
```

Windows looks to be different further—possibly it's `DeviceIoControl`—but this isn't something I know about.

I can try and do the dance to get Musl in there.

Reply:
fair enough, that sounds okay actually, `System.swift` is also very much in the guts. Happy for you to leave the `#if` if that makes it easier